v20.2/performance-benchmarking-with-tpcc-local.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Performance Benchmarking with TPC-C
+summary: Learn how to benchmark CockroachDB against TPC-C 13k on 3 nodes on your laptop
+toc: true
+toc_not_nested: true
+redirect_from:
+- performance-benchmarking-with-tpc-c-10-warehouses.html
+---
+
+This page shows you how to reproduce [CockroachDB's TPC-C performance benchmarking results](performance.html#scale) on commodity AWS hardware. Across all scales, CockroachDB can process tpmC (new order transactions per minute) at near maximum efficiency. Start by choosing the scale you're interested in:
+
+<div class="filters filters-big clearfix">
+  <button class="filter-button current"><strong>Local</strong></button>
+  <a href="performance-benchmarking-with-tpcc-small.html"><button class="filter-button">Small</button></a>
+  <a href="performance-benchmarking-with-tpcc-medium.html"><button class="filter-button">Medium</button></a>
+  <a href="performance-benchmarking-with-tpcc-large.html"><button class="filter-button">Large</button></a>
+</div>
+
+| Workload | Cluster size                       | Warehouses | Data size |
+|----------+------------------------------------+------------+-----------|
+| Local    | 3 nodes on your laptop             | 10         | 2 GB      |
+| Small    | 3 nodes on `c5d.4xlarge` machines  | 2500       | 200 GB    |
+| Medium   | 15 nodes on `c5d.4xlarge` machines | 13,000     | 1.04 TB   |
+| Large    | 81 nodes on `c5d.9xlarge` machines | 140,000    | 11.2 TB   |
```

Comment:
@aayushshah15 or @adityamaru The other table has `12TB`. Which one is accurate? I want to be consistent here and in the blog post as well

Reply:
I have updated this to 11.2 based on a comment from Aayush (12TB was for 150k, 11.2 for 140k)